### ISSUE TITLE
Support for other values for "front" vector

### ISSUE THREAD

**Issue by unframework** [regl-project/regl-camera#14]
Nice library, great work! However, I seem to have hit a snag in my usage.

I am setting up a view where Z-axis points up, and X/Y describe a horizontal plane that the viewer is looking at from above. I specified the `up` vector to be `[0, 0, 1]`, correspondingly, but that's when mouse rotation stopped working: both horizontal and vertical mouse movement only change the "theta" angle. Seems like that is because the Z-axis is hardcoded to also be the "front"? Where I need it to be `[0, 1, 0]` instead.

In other words, since the options accept an `up` vector, it would be nice to set the `front` as well.

Thanks!

**Comment by gre** [comment]
definitely looking for this as well as a `right` vector, but these probably can be computed from other values? not sure how
